manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/how-to-configure-alerts.md

```diff
@@ -111,7 +115,45 @@ data0
 > **Note**:
 > - The `noRecoveries()` method suppresses recovery alerts, ensuring only critical alerts are sent.
 
+### Subscribing to OPC UA Alerts using Sample OPCUA Subscriber
+
+1. Expose the port `4840` of `ia-opcua-server` service in [docker-compose.yml](https://github.com/open-edge-platform/edge-ai-suites/blob/main/manufacturing-ai-suite/wind-turbine-anomaly-detection/docker-compose.yml#L183) as below
```

Comment:
Let's replace the link with [docker-compose.yml](https://github.com/open-edge-platform/edge-ai-suites/blob/main/manufacturing-ai-suite/wind-turbine-anomaly-detection/docker-compose.yml#L183) with `edge-ai-suites/manufacturing-ai-suite/wind-turbine-anomaly-detection/docker-compose.yml` to be consistent with how it's done at all other places